.github/workflows/tests.yaml

```diff
@@ -28,8 +30,8 @@ jobs:
     steps:
     - uses: actions/checkout@v1
 
-    - name: Switch Xcode 12
-      run: sudo xcode-select -switch /Applications/Xcode_12.2.app
+    - name: Switch to Xcode 12.5.1
+      run: sudo xcode-select -s /Applications/Xcode_12.5.1.app
 
     - name: Install iOS ${{ matrix.sdk }}
       run: xcversion simulators --install="iOS ${{ matrix.sdk }}"
```

Comment:
I can't comment on the line, but it's above on line 27/28:

Should we bump the iOS 14 version to 14.5?

Reply:
I think SPOS is doing that so probably should match it

Reply:
Makes everything easier (e.g., will likely have matching sims installed locally)

Reply:
- [ ] Use 14.5 simulator